src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -471,6 +469,7 @@ export default function SubdomainEditor() {
           previewChildNodes?.[index] &&
           (child as any).tagName !== "style"
         ) {
+          if (child.position.start.line > view.state.doc.lines) return
```

Comment:
Can you help me explain what this line does? 🤯

Reply:
You can try deleting the entire article at once. I don't know what happened, the number of lines parsed will be more than the number of lines in the editor. 🤨

Reply:
This may be because the `computedPosition` method was run before the article tree refreshed (because of debouncing).

Reply:
Is this the operation? I didn't find any problems on my computer.

https://user-images.githubusercontent.com/8266075/233962262-149a9c06-16eb-49b9-b2c9-7862ae19331d.mp4

Reply:
> Is this the operation? I didn't find any problems on my computer.
> 
>  4.24.mp4

I think it should be reproducible like this:

The editor needs to have a scrollbar;
The scrollbar is not at the top;
Deleting content causes the scrollbar to change